packages/component-library/src/LineChart/LineChart.js

```diff
@@ -87,6 +81,7 @@ const LineChart = ({
 
       <VictoryChart
         domain={chartDomain}
+        padding={{ left: 75, right: 50, bottom: 50, top: 50 }}
```

Comment:
Does it makes sense to move this up into the theme at some point?

Reply:
Actually, this one varies by chart so I think keeping it here is ok.